internal/controller/seaweed_maintenance.go

```
@@ -17,6 +17,7 @@ func (r *SeaweedReconciler) maintenance(m *seaweedv1.Seaweed) (done bool, result
 
 	// this step blocks since the operator can not access the masters when running from outside of the k8s cluster
 	sa := swadmin.NewSeaweedAdmin(masters, "", nil, ioutil.Discard)
+	defer sa.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `sa.Close` is not checked

(errcheck)